### PR TITLE
visitor index without bd

### PR DESCRIPTION
**Pull request by PatrickCAlmeida**
visitor index, controller and view without bd seed

### CODE REVIEW

File: app/views/spaces/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<%= render "shared/navbar" %>
+<%= render "shared/herobanner" %>
+<div>
+  <% @users.each do |user| %>
```
Comment (Kirilbt) [inline app/views/spaces/index.html.erb:4]: Should be changed to spaces

File: app/controllers/spaces_controller.rb [hunk 2]
```diff
@@ -1,2 +1,7 @@
 class SpacesController < ApplicationController
+
+  def index
+    @spaces = Space.all
+    @users = User.all
```
Comment (Kirilbt) [inline app/controllers/spaces_controller.rb:5]: Remove @users

**Review by Kirilbt — CHANGES_REQUESTED** [review]

File: app/controllers/spaces_controller.rb (hunk 2 above)
Comment (PatrickCAlmeida) [reply app/controllers/spaces_controller.rb]: Done. Can you check pls?

**Review by Kirilbt — APPROVED** [review]
All good

**Pull request merged**